public/js/main.js

```diff
@@ -645,6 +645,37 @@ function alignRightHeaders() {
     });
 }
 
+function addSingleQuoteToCsvIfNeeded(str) {
+  const specialChars = ['=', '+', '-', '@', '\t', '\r'];
+  if (specialChars.some(char => str.startsWith(char))) {
+    return "'" + str;
+  } else {
+    return str;
+  }
+}
+
+/**
+ * Patch the Tabulator CSV downloader logic to add a single quote to values starting with special characters
+ * This fixes the issue where Excel would interpret these values as formulas, creating a security risk
+ */
+function patchTabulatorCsvDownloader() {
+  const getFieldValueCopy = Tabulator.prototype.moduleBindings.download.prototype.getFieldValue;
+  Tabulator.prototype.moduleBindings.download.prototype.getFieldValue = function(columnName,rowData) {
+    const context = this;
+    let value = getFieldValueCopy.call(context, columnName, rowData);
+    switch (value === undefined ? "undefined" : typeof value) {
```

Comment:
iirc, typeof null === “object”, but typeof undefined === “undefined”.
so you might want to change this to :
```suggestion
    switch (value === null ? "null" : typeof value) {
```

Reply:
deleted, it was unnecessary